algorithm/cgal/fair.js

```diff
@@ -3,9 +3,23 @@ import { fromCgalGeometry, withCgalGeometry } from './cgalGeometry.js';
 
 import { ErrorZeroThickness } from './error.js';
 
-export const fair = (inputs, count, density = 0) =>
+export const fair = (
```

Comment:
To what extent are the default parameters OK? I worry a little bit about folks getting started being intimidated by tuning things

Reply:
Yeah, that'll take a little experimentation to figure out -- but hopefully there should be only one principle knob.